app/components/App/App.js

```diff
@@ -18,7 +17,9 @@ class App extends Component {
             <Route exact path='/' component={Play} />
             <Route exact path='/instructions' component={Instructions}/>
             <Route path='/learn' component={Learn} />
-            <p className='copyright-notice'>&#169; 2017 Leta Keane</p>
+            <footer>
+              <p className='copyright-notice'>&#169; 2017 Leta Keane</p>            
+            </footer>
```

Comment:
yeah so I would make a `Footer` component and do something like: 
```
const App = () => {
  <Router>
    <section>
      <Header />
        ...some routes here
      <Footer />
    </section>
  </Router>
}
```

Also consider using the `Switch` component from `React-Router`
https://reacttraining.com/react-router/web/api/Switch